gcalcli/gcal.py

```diff
@@ -1120,9 +1120,9 @@ def _search_for_events(self, start, end, search_text):
 
     def _DeclinedEvent(self, event):
         if 'attendees' in event:
-            attendee = [a for a in event['attendees']
-                        if a['email'] == event['gcalcli_cal']['id']][0]
-            if attendee and attendee['responseStatus'] == 'declined':
+            attendees = [a for a in event['attendees']
+                        if a['email'] == event['gcalcli_cal']['id']]
```

Comment:
You're one space short here, because we want to line up `if` with what's inside the comprehension.